src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -135,6 +145,20 @@ class VertexOrchestrator(BaseOrchestrator, GoogleCredentialsMixin):
     network: Optional[str] = None
     synchronous: bool = False
 
+    cpu_limit: Union[int, str, None] = None
+    memory_limit: Optional[str] = None
+    node_selector_constraint: Optional[
+        Literal[
+            "NVIDIA_TESLA_A100",
+            "NVIDIA_TESLA_K80",
+            "NVIDIA_TESLA_P4",
+            "NVIDIA_TESLA_P100",
+            "NVIDIA_TESLA_T4",
+            "NVIDIA_TESLA_V100",
+        ]
+    ] = None
```

Comment:
It took me some trial and error until I realized that I was using a region without any available gpu's - would you be willing to add some information to the docstrings regarding these fields? I would include links like this: https://cloud.google.com/compute/docs/gpus/gpu-regions-zones. At a later stage we'll then expand on these docstrings to give users all the relevant information

Reply:
Added to the docstring.